internal/brokers/examplebroker/ExampleBroker

```diff
@@ -0,0 +1,8 @@
+# Add this to /etc/authd/broker.d to configure the ExampleBroker
+name = ExampleBroker
+brand_icon = /usr/share/backgrounds/warty-final-ubuntu.png
+
+[dbus]
+name = com.ubuntu.auth.ExampleBroker
```

Comment:
Check the spec, we suffixed with **1** for the first API calls versionning. But thinking about it, we can avoid it for v0 and only do it once we break it the first time. Tell me what you think.

Reply:
I feel like there's no need to add a one if there's nothing below it (i.e. `0.x` or something), so I think it's probably better to leave it empty for now. But I don't trust my thoughts on this regard for now, so whatever you decide it's fine with me.